scripts/install_radar_deps.py

```diff
@@ -474,6 +481,7 @@ def main():
     parser.add_argument("--python-version", help="The version of Python to use for the installation. Default 3.9",
                         default='3.9')
     parser.add_argument("--upgrade-to-v06", help="Is this to upgrade from Borealis v0.5 to v0.6?", action="store_true")
+    parser.add_argument("--no-cuda", help="Do not install CUDA and cupy libraries.", action="store_true")
```

Comment:
shouldnt this be store_false and have default true, the logic seems wrong to me with the install_borealis_env argument as default true, this with no default, etc

Reply:
You are definitely right, thanks for catching that